### PR TITLE
Improves readme

### PR DESCRIPTION
**Pull request by TomKlotzPro**
## Describe your change

Improves readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,363 +1,56 @@
...
-    container: '#hits',
-    templates: {
-      item: document.getElementById('hit-template').innerHTML,
-      empty: "We didn't find any results for the search <em>\"{{query}}\"</em>"
-    }
-  })
-);
-
-search.start();
+objects = $index.search('Fo')
```
Comment (nunomaduro) [inline README.md]: The `$` is php code no? Don't hesitate to run the example on the playground first.

File: README.md [hunk 2]
```diff
@@ -1,363 +1,56 @@
...
+  <a href="http://stackoverflow.com/questions/tagged/algolia" target="_blank">Stack Overflow</a>  •
+  <a href="https://github.com/algolia/algoliasearch-client-ruby/issues" target="_blank">Report a bug</a>  •
+  <a href="https://www.algolia.com/support" target="_blank">Support</a>
+</p>
 
+## ✨ Features
 
-If you are a **Ruby on Rails** user, you are probably looking for the [algoliasearch-rails](https://github.com/algolia/algoliasearch-rails) gem.
+- Thin & minimal low-level HTTP client to interact with Algolia's API
+- Supports Ruby `^1.8.7`.
```
Comment (nunomaduro) [inline README.md]: Where did you got this information?

Comment (TomKlotzPro) [reply README.md]: From the ruby playground

File: README.md (hunk 1 above)
Comment (TomKlotzPro) [reply README.md]: Yes ! Sorry, i’ve tested the wrong snippet

File: README.md (hunk 2 above)
Comment (nunomaduro) [reply README.md]: What do you mean buddy?

**Comment by nunomaduro** [comment]
Thanks @TomKlotzPro

**Pull request merged**